flowey/flowey_lib_hvlite/src/download_openvmm_vmm_tests_vhds.rs

```diff
@@ -440,11 +436,15 @@ Otherwise, press `ctrl-c` to cancel the run.
         Ok(())
     }
 }
+
+#[allow(unused)]
 enum AzCopyAuthMethod {
     /// Pull credentials from the Azure CLI instance running the command.
     AzureCli,
     /// Print a link to stdout and require the user to click it to authenticate.
     Device,
+    /// Files do not need authentication
+    None,
```

Comment:
Why not just have the fn below take Option<AzCopyAuthMethod>? None isn't really a method